crates/lib/sig-check/rsa/Nargo.toml

```diff
@@ -5,6 +5,9 @@ authors = ["Theo Madzou"]
 compiler_version = ">=0.36.0"
 
 [dependencies]
-rsa = { tag = "main", git = "https://github.com/zkpassport/noir_rsa.git", directory = "lib" }
-bignum = { tag = "main", git = "https://github.com/noir-lang/noir-bignum.git" }
+# rsa = { tag = "main", git = "https://github.com/zkpassport/noir_rsa.git", directory = "lib" }
+# rsa = { tag = "mc/update-to-0.4.0", git = "https://github.com/iAmMichaelConnor/noir_rsa.git", directory = "lib" }
+rsa = { path = "../../../../../noir_rsa/lib" } # https://github.com/iAmMichaelConnor/noir_rsa/blob/mc/update-to-0.4.0/lib/Nargo.toml but with a Nargo.toml which points the `main` branch of bignum.
```

Comment:
The comment is a lie. The version that compiled was a version of noir_rsa which pointed to a local path of the `main` branch of bignum.